src/enum.ts

```diff
@@ -1,17 +1,22 @@
 import * as vscode from "vscode"
-import { getApiDump, ApiEnum } from "./dump"
+import { ApiEnum, getApiDump } from "./dump"
 
 export class EnumCompletionProvider implements vscode.CompletionItemProvider {
-    enumItems: Promise<vscode.CompletionItem[]>
-    enumNamesAndItems: Promise<{ [name: string]: vscode.CompletionItem[] }>
+    public enumItems: Promise<vscode.CompletionItem[]>
```

Comment:
I generally elude `public`/`private` since I haven't seen it be too useful in practice.

Reply:
I added this in because of TSLint warnings (recommended extension) - I think the TSLint rules should be updated for these cases [And probably also start moving to ESLint]

Reply:
I think I use ESLint since TSLint is being deprecated soon. I recommend staying consistent with the rest of the codebase, and at a later point I might enforce a stricter code style like with ESLint and overhaul everything then.